### ISSUE TITLE
Usage of Django (API) for Backend

### ISSUE THREAD

**Issue by ShehriyarShariq** [CS334SEECS/nust-financial-reward#7]
Sir, can the backend be instead developed in Python using the Django Rest Framework?

**Comment by hasanalikhattak** [comment]
does it run on Github pages

**Comment by ShehriyarShariq** [comment]
Yes, sir. It would be a simple API call. Github pages does have support for AJAX and it allows calls over HTTPS.

**Comment by hasanalikhattak** [comment]
but the page needs to be hosted somewhere else ?

**Comment by ShehriyarShariq** [comment]
The API can be hosted for free on https://www.pythonanywhere.com/ . It also provides free HTTPS support.

**Comment by hasanalikhattak** [comment]
great, you can begin by creating the form first.

**Comment by ShehriyarShariq** [comment]
Great. I can create a documentation on postman for the person who will be doing frontend, so that it's easier to coordinate.
